requests_ntlm/requests_ntlm.py

```diff
@@ -31,9 +34,11 @@ def __init__(self, username, password, session=None):
                 self.domain = '.'
 
         self.domain = self.domain.upper()
-
         self.password = password
 
+        # Used by other python libraries to sign/seal messages with NTLM methods
+        self.session_security = None
```

Comment:
I think we're talking at cross purposes to each other.

My question is, given that requests-ntlm doesn't use this property, *who* is expected to use it and under what circumstances?